credentials-list-directive.js

```diff
@@ -29,6 +31,9 @@ function factory($location, brAlertService, brCredentialService, config) {
     model.state = {
       credentials: {loading: true}
     };
+    model.showHeadline = scope.showHeadline === false ? false : true;
+    model.showPublicIndicator =
+      scope.showPublicIndicator === false ? false : true;
```

Comment:
We want the headline to be shown by default (showHeadline = true).  An optional attribute to the directive is `undefined` which evaluates to `false` which we do not want.  We only want to set showHealine to false if the attribute is explicitly set to false.

Do you think that having the attributes be `brHideFoo` instead of `brShowFoo` and doing a direct assignment would be better?